02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -170,26 +170,285 @@ Here's what has been added to the above since then:
+
+#### Why Not Just Delegate?
+
+> A hypothetical developer says, "If it's so much work, why not just **delegate** the work? This 'somebody' can also be a proxy who's gathering feedback, so that you don't have to be in these discussions."
+>
+> The contributor responds sarcastically, "Oh! Delegation, of course! I hadn't thought about that...."
+>
+> There's actually an assumption going on here: "If 'free' rice means you can take as much as you want, then 'free' labor implies you can take as much as you want." But that isn't how labor works. If you don't pay for labor, you get less.
+>
+> Even if everyone could help, there's still limitations due to coordinating efforts: who does what, when, and where, and all without affecting someone else's work.
+>
+> ~ summary of the 'Why Not Just... Pattern' in [The Hard Parts of Open Source](https://youtu.be/o_4EX4dPppA?t=257)
+
+One shouldn't trust everyone on the internet. (I know, big surprise.)
+
+For example, consider the `event-stream incident`. The maintainer unknowingly gave write-access to a malicious actor. Practically everyone depends on this code. The actor injected code that could be used to steal bitcoins:
+- [initial issue](https://github.com/dominictarr/event-stream/issues/116)
+- [injected code explanation](https://github.com/dominictarr/event-stream/issues/116#issuecomment-441759047)
+- [original maintainer's response](https://github.com/dominictarr/event-stream/issues/116)
+- [a sarcastic blog post summarizing this class of attack](https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5)
+
+New maintainers, once found and properly screened, have to be supported (which takes time and thought), so that they can actually fulfill their role. If they decide to leave soon after starting, then one has wasted time and effort.
+
+(Related interpretation sections: [Knowing whom to trust with write access, defining best workflow procedures, and providing necessary support](https://github.com/chexxor/purescript-documentation-discussion/blob/master/01-Sources-and-Interpretation/All-Interpretations.md#knowing-whom-to-trust-with-write-access-defining-best-workflow-procedures-and-providing-necessary-support))
+
+#### Why Not Just Fund the Language Developers?
+
+Been there, done that. Someone took it down. ([PureScript Open Collective]((https://opencollective.com/purescript))
+
+I (Jordan) don't know why it was taken down. Here's my speculation:
```

Comment:
My point of view on that:

- Fans of PureScript wanted to throw money at the project, perhaps because want to help the project and its goals but don't have time or mental capacity to contribute to the project with code or management time.
- Several people pressed a button to indicate they'd like to pledge money to the "github.com/purescript/purescript" project.
- Several weeks later, someone claimed that project's corresponding OpenCollective page and decided to turn off its public OpenCollective page, likely because there was no clear vision as to how the funds would be spent.

Reply:
That sounds about right. Maybe we should just ask in the #purescript channel?

Reply:
If we do ask there for more information, it would only be to ask why it was turned off. I was the initial pledger and posted on social media to spread the word, so I think we won't find a better source for that info. After thinking about whether we need reason about why it was removed, I do think it's important to have a reason, because it would serve as a data to consider for a future party who wants to collect money to support the PureScript project. I guess it won't be easy to get a solid answer on that, however, because it's possible the person who did that doesn't hang out in the community (because I asked this question around the time it happened and I got no response). So, as a back-up plan, could just assume the most general fact - "A maintainer of the purescript/purescript project did not feel comfortable having money in the PureScript language project."

Reply:
> So, as a back-up plan, could just assume the most general fact - "A maintainer of the purescript/purescript project did not feel comfortable having money in the PureScript language project."

That sounds like the best course of action. 

If a person did not answer you when you asked that, either they did not see it, or they did see it and didn't want to provide an answer. I'm not sure which one is correct, but if it's the latter, then I wonder whether they would answer the question even now.

So, let's just use your rendering and note that this is probably true but still unconfirmed.

Reply:
I recall seeing this resolved in a commit.

Reply:
Hmm... It was resolved in an earlier commit. I imagine the force push screwed up something. Sorry about that (again).